### DISCUSSION TITLE
why libcurl no cve?

### DISCUSSION THREAD

**Discussion by chenjianquan7** [DependencyTrack/dependency-track#3147] [General]
![image](https://github.com/DependencyTrack/dependency-track/assets/26828709/d8b97e23-0d48-4611-9dc5-7b3109fcfd11)
![image](https://github.com/DependencyTrack/dependency-track/assets/26828709/44e520e6-2ccc-4c2c-a256-541357431e95)
libcurl 7.65.1 no cve ,why

**Comment by nscuro** [comment]
As explained in the issue you raised earlier (https://github.com/DependencyTrack/dependency-track/issues/3138#issuecomment-1781881031), the CPE you provided does not match any entries in the NVD: https://nvd.nist.gov/products/cpe/search/results?namingFormat=2.3&keyword=cpe%3A2.3%3Aa%3Alibcurl%3Alibcurl%3A7.65.1%3A*%3A*%3A*%3A*%3A*%3A*%3A*

There is no CPE with `libcurl` in the `vendor` attribute of the CPE. If you don't know the correct `vendor`, you can use `*`, which will many *any* vendor in the vulnerability database.

Assuming you're running Dependency-Track 4.9.0, `cpe:2.3:a:*:libcurl:7.65.1:*:*:*:*:*:*:*` will give you results. I explained this in your original issue. Please do not create new issues or discussions for the same thing.
